test/powershell/Modules/Microsoft.PowerShell.Core/Job.Tests.ps1

```diff
@@ -243,3 +243,16 @@ Describe "Ampersand background test" -Tag "CI","Slow" {
         }
     }
 }
+
+Describe "Start-Job with -PSVersion parameter" -Tag "CI" {
+
+    It "Verifies that -PSVersion is not supported except for version 5.1" -Skip:(-not $IsWindows) {
```

Comment:
If we aren't doing a different error for non-Windows, then this test should also pass on non-Windows, right?